blocks/job-list/job-list.css

```diff
@@ -0,0 +1,63 @@
+.job-list {
+  --color-text: var(--spectrum-gray-900);
+  --color-border: var(--spectrum-gray-400);
+  margin-block: 2rem;
+}
+
+.job-list__department {
+  font-size: var(--spectrum-heading-size-s);
+  margin-block: 0 2rem;
+  color: var(--color-text);
+
+  @media (min-width: 48rem) {
+    font-size: var(--spectrum-heading-size-m);
+  }
+
+  @media (min-width: 80rem) {
+    font-size: var(--spectrum-heading-size-s);
+  }
+}
+
+.job-list__listings {
+  display: grid;
+  gap: 0;
+
+  & .job-listing:first-child {
+    border-top: none;
+  }
+
+  & .job-listing:last-child {
+    border-bottom: none;
+  }
+}
+
+@media (min-width: 48rem) {
+  .job-list__listings .job-listing::after {
+    display: none;
+  }
+}
+
+@media (min-width: 80rem) {
+  .job-list__listings {
+    display: grid;
+    grid-template-columns: repeat(2, minmax(0, 32rem));
+    gap: 2rem;
+  }
+
+  .job-list__listings .job-listing {
+    border: none;
+  }
+}
+
+@media (min-width: 105rem) {
+  .job-list {
+    display: grid;
+    grid-template-columns: minmax(20rem, 25rem) 1fr;
+    gap: 4rem;
+  }
+
+  .job-list__department {
+    margin-block: 0;
+    top: 2rem;
+  }
+}
```

Comment:
When there is only one Job Listing in a Job List, there should be no borders on the item. It is unclear to me if that is the current functionality. 

Can we remove the borders completely from the Job Listing and apply them only at the Job List level, and add an example in Sharepoint with some documentation around this fact?

We should also add a little more documentation on using the component as well, noting that a Job List/Job Listing are not tied together, and what type of information they can use. 

The Job Listing will be used on its own outside of the Job List, so we also need to test those styles match the design (I think just no borders) check the pre-footers). And that the documentation states this.

Reply:
Yep we can do that, we'll just have to keep in mind these will need to be added back in when we use the job listing styles in the 3 up layout design.

Reply:
ohhh yeah because those look like a link list on small screens